README.md

```diff
@@ -0,0 +1,91 @@
+# bootstrap-ubuntu-rpi3
+
+This role sets up a Raspberry Pi 3 SD card with a minimal Ubuntu configuration for use with Ansible.
+Only a SSH server is installed along with Python.
+Also, the network is configured with a static IP address.
+
+This role can not run on the Raspberry Pi directly, instead it's supposed to be run on the control machine.
+The SD card must be inserted into the control machine and this role will install everything on it.
+While doing this, qemu-user-static is used.
+It must be installed and configured in the binfmt.
+
+This role is not capable of updating an existing Raspberry Pi without reinstalling.
+
+You need internet access on the control machine to download the kernel, the firmware, and Ubuntu.
+
+You need internet access on the server to clone Vault.
+
+## Requirements
+
+A dpkg- or pacman-based Linux distribution on the host machine.
+qemu-user-static must be configured in binfmt.
+
+On Arch, you also need the Ubuntu keyring package from the AUR.
+
+## Role Variables
+
+#### Global
+
+| Name               | Required           | Default | Description                                                                                   |
+|--------------------|:------------------:|---------|-----------------------------------------------------------------------------------------------|
+| `global_cache_dir` | :heavy_check_mark: |         | Path where files are downloaded and extracted. Also, the SD card is mounted in a subdirectory |
+| `global_cores`     | :heavy_check_mark: | `1`     | Amount of CPU cores in the control machine to build with                                      |
```

Comment:
why do you have a default if the var is mandatory? Same with the other vars below.